Dockerfile

```diff
@@ -1,6 +1,9 @@
-FROM openjdk:8-jdk
+FROM centos:7
 
-RUN apt-get update && apt-get install -y git curl && rm -rf /var/lib/apt/lists/*
+RUN yum install -y java-1.8.0-openjdk-headless
```

Comment:
Just include this in the run step below. There is no need to separate this package installation here.